ingestion_tools/scripts/data_validation/conftest.py

```diff
@@ -98,6 +98,64 @@ def get_voxel_spacings_set(voxel_spacing_files: List[str]) -> List[str]:
     return list(set(voxel_spacings))
 
 
+def dataset_run_tiltseries_combinations(
+    fs: S3Filesystem,
+    bucket: str,
+    dataset_run_combinations: List[Tuple[str, str]],
+) -> List[Tuple[str, str, str]]:
+    files = []
+
+    def get_tiltseries_files(dataset_run_tuple) -> List[Tuple[str, str, str]]:
+        glob_str = f"s3://{bucket}/{dataset_run_tuple[0]}/{dataset_run_tuple[1]}/TiltSeries/*"
+        files = []
+        for item in fs.glob(glob_str):
+            tiltseries_dir = os.path.basename(item)
+            files.append((*dataset_run_tuple, tiltseries_dir))
+        return files
+
+    with concurrent.futures.ThreadPoolExecutor() as executor:
+
+        files_lists = executor.map(
+            get_tiltseries_files,
+            dataset_run_combinations,
+        )
+        files = [
+            res_tuple
+            for res_list in files_lists
+            for res_tuple in res_list
+        ]
+
+    return files
+
+def dataset_run_tomogram_combinations(
```

Comment:
We had some tests that assumed a single tomogram per run. This lets us update them to support multiple tomos per run.